examples/kafka_spark_demo/feature_repo/streaming_features_tutorial.ipynb

```diff
@@ -0,0 +1,877 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "22/06/17 11:33:13 WARN ResolveWriteToStream: spark.sql.adaptive.enabled is not supported in streaming DataFrames/Datasets and will be disabled.\n"
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "df columns: Index(['event_timestamp', 'created', 'conv_percentage', 'acc_percentage'], dtype='object')\n",
+      "df size: 0\n",
+      "df preview:\n",
+      "Empty DataFrame\n",
+      "Columns: [event_timestamp, created, conv_percentage, acc_percentage]\n",
+      "Index: []\n"
+     ]
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/Users/felixwang/feast/sdk/python/feast/stream_feature_view.py:98: RuntimeWarning: Stream Feature Views are experimental features in alpha development. Some functionality may still be unstable so functionality can change in the future.\n",
+      "  warnings.warn(\n"
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "df columns: Index(['event_timestamp', 'created', 'conv_percentage', 'acc_percentage'], dtype='object')\n",
+      "df size: 20\n",
+      "df preview:\n",
+      "              event_timestamp                          created  \\\n",
+      "driver_id                                                        \n",
+      "1001      2024-02-26 08:00:00 2022-06-17 18:33:31.261210+00:00   \n",
+      "1002      2024-02-26 08:00:00 2022-06-17 18:33:31.261210+00:00   \n",
+      "1003      2024-02-26 08:00:00 2022-06-17 18:33:31.261210+00:00   \n",
+      "1004      2024-02-26 08:00:00 2022-06-17 18:33:31.261210+00:00   \n",
+      "1005      2024-02-26 08:00:00 2022-06-17 18:33:31.261210+00:00   \n",
+      "\n",
+      "           conv_percentage  acc_percentage  \n",
+      "driver_id                                   \n",
+      "1001             81.257027       60.436976  \n",
+      "1002             85.917443       89.079225  \n",
+      "1003             23.591906       90.642333  \n",
+      "1004             33.968881        6.300516  \n",
+      "1005              4.525895       70.851427  \n",
+      "writing row\n"
+     ]
+    }
+   ],
+   "source": [
+    "query = ingest_stream_feature_view(store, \"driver_hourly_stats_stream\", spark_config)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Next we retrieve features."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 14,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "acc_percentage  :  [60.43697738647461]\n",
+      "conv_percentage  :  [81.25702667236328]\n",
+      "driver_id  :  [1001]\n"
```

Comment:
IMO it would be great if we could show the event_timestamp or whatever to show how fresh the features are. Dunno if this is currently possible but just spit balling :)

Reply:
great idea, added timestamps!

Reply:
do timestamps work? I thought we return always now() for the time stamps so it's kind of misleading (pretty sure there's a GitHub issue for this too)

Reply:
😭